hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/realtime/AbstractRealtimeRecordReader.java

```diff
@@ -164,11 +165,26 @@ protected static String arrayWritableToString(ArrayWritable writable) {
    * columns
    */
   public static Schema generateProjectionSchema(Schema writeSchema, List<String> fieldNames) {
+    /**
```

Comment:
Without these - select queries did not work for trips schema. But, this has implication on how hoodie can play well with query engines.